server/utils/middleware/filterTag.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * This middleware function removes any empty string
+ * in the tags attribute from the user's request body.
+ * It calls the next() function once it is done
+ *
+ * @param {object} req the request object that contains the request
+ * made by the user
+ * @param {object} res  an object used to send response to the
+ * client
+ * @param {function} next used to send control to the next middleware
+ * @returns {void} this method does not return any value
+ */
+const filterTag = (req, res, next) => {
+  let { tags } = req.body.article;
```

Comment:
@chidioguejiofor can you make me understand why you're using `let` here ?

Reply:
I am using let because I reassigned that variable if you look down the code